.github/workflows/deploy-changed-cf.yaml

```diff
@@ -296,7 +296,7 @@ jobs:
             exit 1
           fi
           SETUP_ROLES_CMD="
-            PGPASSWORD=${AURORA_${{ steps.setup-rds-env.outputs.uppercase }}_PASSWORD} psql \
+            PGPASSWORD='${AURORA_${{ steps.setup-rds-env.outputs.uppercase }}_PASSWORD}' psql \
```

Comment:
This is a fix that already worked for the `deploy-staging` step `setup-rds-roles`, so it shoudl work for this too